### PR TITLE
Development (awaiting approval)

### PR DESCRIPTION
**Pull request by Legend-of-iPhoenix**
# Describe your changes
Adds styling and main HTML by @jcgter777, adds bolding, italics and underline stuff by me.

### CODE REVIEW

**Comment by josheewa** [comment]
Okay cool. What do I need to do now?

**Review by josheewa — APPROVED** [review]
👍

**Comment by Legend-of-iPhoenix** [comment]
I'll merge it once the checks get approved.

**Pull request merged**
